src/components/table/partial-styles/tabulator-custom-styles.scss

```diff
@@ -1,32 +1,87 @@
 @use '../../../style/mixins';
 @use '../../../style/functions';
 
+.interactive-feedback {
+    // This is a div, injected by us into all rows.
+    // We use it to visualize interactive feedback.
+    position: absolute;
+    inset: 0;
+    min-height: 0.125rem;
+
+    // We could use `@include mixins.is-flat-clickable()`,
+    // But the "arguments" of the mixin would make the CSS for
+    // even and odd row colors and background color very complicated.
+    // So instead, I've copy/pasted the useful parts of the mixin here
+    transition: color 0.2s ease, background-color 0.2s ease,
```

Comment:
But do we need transform here? It never really transforms (as in scales/rotates etc.)?

Reply:
when the row is pressed (`active`), it slightly moves down. that's the transform